charts/app-config/templates/external-secrets/static/google-tag-manager-preview.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if not (eq $.Values.govukEnvironment "production") }}
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: static-google-tag-manager-preview
+  labels:
+    {{- include "app-config.labels" . | nindent 4 }}
+  annotations:
+    kubernetes.io/description: >
+      gtm_auth and gtm_preview values to enable the Google Tag Manager preview
+      facility in non-production Static.
```

Comment:
Is it the case that this secret will no exist in Production?

Reply:
That's right — doesn't exist in current Production and won't need to in ours either.

Reply:
It's for testing changes to Google Tag Manager configuration without applying them to the live website, hence it's not applicable to production (because there wouldn't be any point in "previewing" changes on the live website, because that's the same as just making the changes live).